packages/itwinui-css/backstop/tests/table.html

```diff
@@ -4616,22 +4616,54 @@ <h2>Empty state</h2>
         }
       };
 
+      const updateRowSelectionCount = (table) => {
+        const checkboxes = [...table.querySelectorAll('.iui-table-body .iui-table-row .iui-checkbox:not(:disabled)')];
+        const numBoxesChecked = checkboxes.filter((checkbox) => checkbox.checked).length;
+        const leftPaginator = table.querySelector('.iui-table-paginator .iui-left');
+        const paginatorCheckboxCount = leftPaginator.querySelector('.iui-table-paginator-selection-count');
+        if (!numBoxesChecked) {
+          if (paginatorCheckboxCount) {
+            leftPaginator.removeChild(paginatorCheckboxCount);
+          }
+          return;
+        }
+        const paginatorCheckboxCountText = numBoxesChecked === 1 ? '1 row selected' : `${numBoxesChecked} rows selected`;
```

Comment:
Would it be better to have "0 rows selected" versus the text disappearing? The text disappearing is very nice and clean, but I think showing "0 rows selected" gives more information to the user.

Reply:
I think we should not show "0 rows selected". It gives 0 useful information and might add to visual clutter.

Reply:
I was thinking that if a user can't see all rows at the same time, it tells them that there are no rows they can't see that are selected. If you're watching for it, it's easy to understand that no label means nothing selected. But if you're not watching it, there's nothing explicitly saying that nothing is selected.

Reply:
We initially went with removing the text when there is no selection because that is what [this demo (our inspiration) did.](https://codesandbox.io/s/fervent-sutherland-37vruc)

Reply:
Gotcha. I don't think the current way is bad; just wanted to point out another way